File: app/src/main/java/com/empreendapp/collev/ui/system/IntroActivity.java
package com.empreendapp.collev.ui.system;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.ImageView;
import android.widget.RadioButton;
import android.widget.RadioGroup;
import android.widget.TextView;

import com.empreendapp.collev.R;

public class IntroActivity extends AppCompatActivity {
    ImageView imgLogo;
    TextView tvBoasVindas;
    RadioGroup rgSlide;
    RadioButton bSlide1, bSlide2, bSlide3;



    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_intro);
        initViews();
    }

    private void initViews() {
        imgLogo = (ImageView) findViewById(R.id.imgLogo);
        tvBoasVindas = (TextView) findViewById(R.id.tvBoasVindas);
        rgSlide = (RadioGroup) findViewById(R.id.rgSlide);
        bSlide1 = (RadioButton) findViewById(R.id.bSlide1);
        bSlide2 = (RadioButton) findViewById(R.id.bSlide2);
        bSlide3 = (RadioButton) findViewById(R.id.bSlide3);

        View.OnClickListener onClickListener = new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                RadioButton rButton = (RadioButton) v;
                rgSlide.clearCheck();
                rButton.setChecked(true);
            }
        };

        bSlide1.setOnClickListener(onClickListener);
        bSlide2.setOnClickListener(onClickListener);
        bSlide3.setOnClickListener(onClickListener);
    }


}

File: app/src/main/java/com/empreendapp/collev/ui/system/IntroActivity.kt
package com.empreendapp.collev.ui.system

import androidx.appcompat.app.AppCompatActivity
import android.widget.TextView
import android.widget.RadioGroup
import android.widget.RadioButton
import android.os.Bundle
import com.empreendapp.collev.R
import android.content.Intent
import android.os.Handler
import android.view.View
import android.widget.ImageView
import com.daimajia.androidanimations.library.YoYo
import com.daimajia.androidanimations.library.Techniques
import com.empreendapp.collev.util.DefaultLayout.Companion.setStatusBarBorderRadiusWhite

class IntroActivity : AppCompatActivity() {
    //IntroActivityBinding binding;
    var imgLogo: ImageView? = null
    var tvBoasVindas: TextView? = null
    var rgSlide: RadioGroup? = null
    var bSlide1: RadioButton? = null
    var bSlide2: RadioButton? = null
    var bSlide3: RadioButton? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_intro)
        setStatusBarBorderRadiusWhite(this)
        initViews()
        showIntroAnimate()
    }

    private fun initViews() {
        imgLogo = findViewById<View>(R.id.imgLogo) as ImageView
        tvBoasVindas = findViewById<View>(R.id.tvBoasVindas) as TextView
        rgSlide = findViewById<View>(R.id.rgSlide) as RadioGroup
        bSlide1 = findViewById<View>(R.id.bSlide1) as RadioButton
        bSlide2 = findViewById<View>(R.id.bSlide2) as RadioButton
        bSlide3 = findViewById<View>(R.id.bSlide3) as RadioButton

        val onClickListener = View.OnClickListener { v ->
            val rButton = v as RadioButton
            rgSlide!!.clearCheck()
            rButton.isChecked = true
        }

        bSlide1!!.setOnClickListener(onClickListener)
        bSlide2!!.setOnClickListener(onClickListener)
        bSlide3!!.setOnClickListener(onClickListener)
    }

    private fun showIntroAnimate() {
        imgLogo!!.visibility = View.VISIBLE

        //play animate logo
        YoYo.with(Techniques.FadeIn).duration(1600).repeat(0).playOn(imgLogo)

        //contar tempo (700ms)
        val handler = Handler()
        val r = Runnable {
            tvBoasVindas!!.visibility = View.VISIBLE

            //play animate logo
            YoYo.with(Techniques.FadeIn).duration(800).repeat(0).playOn(tvBoasVindas)
            val handler2 = Handler()
            val r2 = Runnable {
                startActivity(Intent(this, LoginActivity::class.java))
                finish()
            }
            handler2.postDelayed(r2, 1600)
        }
        handler.postDelayed(r, 2000)
    }
}